src/OrbitGl/CallstackThreadBar.h

```diff
@@ -51,13 +51,22 @@ class CallstackThreadBar : public ThreadBar {
 
  private:
   void SelectCallstacks();
-  [[nodiscard]] std::string SafeGetFormattedFunctionName(
-      const orbit_client_data::CallstackInfo& callstack, size_t frame_index,
-      int max_line_length) const;
-  [[nodiscard]] std::string FormatCallstackForTooltip(
-      const orbit_client_data::CallstackInfo& callstack, int max_line_length = 80,
-      int max_lines = 20, int bottom_n_lines = 5) const;
 
+  struct ModuleAndFunctionNameToFormat {
```

Comment:
Nit: I'd prefer `UnformattedModuleAndFunctionName`, I found it a bit hard to understand the meaning of this at first.